docs/systems/servers.md

```diff
@@ -60,6 +60,11 @@ OCDS Kingfisher is the data access and analysis platform for OCDS data.
 
 There is a hosted version of the software, called [Hosted Kingfisher](https://ocdskingfisher.readthedocs.io/en/latest/). 
 
+## Standard Search
+
+`live.standard-search.opencontracting.uk0.bigv.io` hosts the standard search service, 
+which is called directly from the web browser by [javascript hosted in the docs theme](https://github.com/open-contracting/standard_theme/blob/open_contracting/standard_theme/static/js/search.js).
```

Comment:
1. Properly capitalize JavaScript
2. docs theme -> standard_theme repository (for clarity)